src/components/PostSessionFeedback.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+import Satisifed from "@/components/icons/Satisfied.vue";
+import Neutral from "@/components/icons/Neutral.vue";
+import Dissatisfied from "@/components/icons/Dissatisfied.vue";
+
+export default {
+    props: {
+        name: {
+            type: String,
+            required: true,
+        }
+    },
+    components: {
+        Satisifed,
+        Neutral,
+        Dissatisfied,
+    },
+    emits: ["satisfied", "neutral", "dissatisfied"],
+}
+</script>
+
+<template>
+    <div class="centered">
+        <div class="form-horizontal border p-3 rounded thecard">
+
+            <div class="centered">
+                <h2> How was session {{ name }}?</h2>
+            </div>
+
+
+            <div class="centered">
+                <div class="emotions">
+                    <div @click="$emit('satisfied')" class="face">
+                        <Satisifed />
+                    </div>
+                    <div @click="$emit('neutral')" class="face">
+                        <Neutral />
+                    </div>
+                    <div @click="$emit('dissatisfied')" class="face">
+                        <Dissatisfied />
+                    </div>
+                </div>
+            </div>
+
+        </div>
+    </div>
+</template>
+
+<style scoped>
+.centered {
+    display: flex;
```

Comment:
there is a bootstrap class for that - `d-flex`

Reply:
But this is used 3 times, and it's nice to not duplicate it on all the els

Reply:
put it in `main.css`, not `scoped`

Reply:
I'm worried about it being in the global namespace though. If we end up using it everywhere thats fine, but lets not add spooky action at a distance yet.